ceph/rados/core_workflows.py

```diff
@@ -1194,10 +1261,15 @@ def create_erasure_pool(self, name: str, **kwargs) -> bool:
         crush_num_failure_domains = kwargs.get("crush-num-failure-domains", None)
         create_rule = kwargs.get("create_rule", False)
         plugin = kwargs.get("plugin", "jerasure")
-        pool_name = kwargs.get("pool_name")
-        force = kwargs.get("force")
-        profile_name = kwargs.get("profile_name", f"ecp_{name}")
-        rule_name = f"rule_{name}"
+        pool_name = kwargs.get("pool_name", "name")
+        if not pool_name:
+            log.error("No name provided. Exiting")
+            return False
```

Comment:
Will this block ever get executed?

Reply:
Hopefully, it would never reach here, Just added this as a failsafe, so that we can fail tests.